docs/command_line_reference.rst

```diff
@@ -64,6 +64,11 @@ Examples::
 
 Selects the track repository that Rally should use to resolve tracks. By default the ``default`` track repository is used, which is available in the Github project `rally-tracks <https://github.com/elastic/rally-tracks>`_. See the :doc:`track reference </track>` on how to add your own track repositories. ``--track-path`` and ``--track-repository`` as well as ``--track`` are mutually exclusive.
 
+``track-revision``
+~~~~~~~~~~~~~~~~~
```

Comment:
When building the docs, this produces the following warning:

```

``track-revision``
~~~~~~~~~~~~~~~~~
~/Projects/rally/docs/command_line_reference.rst:68: WARNING: Title underline too short.
```

Can you please fix the warning?

Reply:
Fixed